repository-data/application/src/main/resources/hcm-content/content/documents/administration/valuelists/persontitles.yaml

```diff
@@ -0,0 +1,37 @@
+/content/documents/administration/valuelists/persontitles:
```

Comment:
Thanks @Raq-HEE  for the idea. I think it's a nice workaround but since we already have the code written it would be  wasteful to delete, so I would recommend keeping it as is.

The code follows the best practices mentioned by BR around ValueLists and I wrote it in a way that will make new ValueLists to be easily added in the future.

Reply:
Thanks @adaMiclos - agreed. Will be useful for the work I'm doing on NWPS-124.